### DISCUSSION TITLE
IM Display missing in Windows

### DISCUSSION THREAD

**Discussion by junglist724** [ImageMagick/ImageMagick#7399] [Help] [answered]
### ImageMagick version

7.1.1-33

### Operating system, version and so on

Windows 10 Enterprise 22H2, 19045.4529

### Description

I recently upgraded from a rather old version of ImageMagick(7.1.0.57) and when using the installer for 7.1.1-33 I noticed that imdisplay.exe does not exist in the install directory. Has this been removed? I couldn't find any information in the changelog about this.

**Comment by dlemstra** [comment, marked as answer]
You can find more information about why we have do that here: #7246

**Reply by junglist724** [reply]
Thanks for the fast response!
